anaconda.spec.in

```diff
@@ -143,6 +143,8 @@ Requires: python3-pid
 Requires: crypto-policies
 Requires: crypto-policies-scripts
 
+Requires: grub2-common
```

Comment:
I think this should be in install-env-deps (eq. the package that pulls in dependencies needed to do a a full installation run) as otherwise it could get pulled in when just the Anaconda package gets installed, say a an `initial-setup` dependency.

IIRC some people potentially use something else than grub (systemd-boot ? at least there seem to be enough users [to complain](https://www.spinics.net/lists/fedora-devel/msg324206.html) on fedora-devel when its broken by grub specifics). If it adds post-transaction scriptlets, I guess it could actively break something.